topics/terraform/index.md

```diff
@@ -0,0 +1,19 @@
+---
+created_by: Mitchell Hashimoto, HashiCorp
+display_name: Terraform
+github_url: https://github.com/hashicorp/terraform
+logo: terraform.png
+related: hashicorp, hashicorp-terraform, infrastructure, infrastructure-as-code, terraform-configurations, terraform-module, terraform-modules, terraform-providers
+released: July 28, 2014
+short_description: An infrastructure as code tool for building, changing, and versioning infrastructure safely and efficiently.
+topic: terraform
+url: https://www.terraform.io
+wikipedia_url: https://en.wikipedia.org/wiki/Terraform_(software)
+---
+Terraform can manage existing and popular service providers as well as custom in-house solutions.
```

Comment:
What about listing one of the service providers as an example, with "providers, such as X, as well"?